src/components/App.tsx

```diff
@@ -776,10 +788,29 @@ class App extends React.Component<AppProps, AppState> {
 
     if ("ResizeObserver" in window && this.excalidrawContainerRef?.current) {
       this.resizeObserver = new ResizeObserver(() => {
+        // compute isMobile state
+        // ---------------------------------------------------------------------
+        const {
+          width,
+          height,
+        } = this.excalidrawContainerRef.current!.getBoundingClientRect();
+        this.isMobile =
+          width < MQ_MAX_WIDTH_PORTRAIT ||
+          (height < MQ_MAX_HEIGHT_LANDSCAPE && width < MQ_MAX_WIDTH_LANDSCAPE);
+        // refresh offsets
+        // ---------------------------------------------------------------------
```

Comment:
can you move this to a function ?

Reply:
seems like an overkill to factor out two statements into a function. Or what are we talking about?

Reply:
should be fine, I generally prefer making code modular when doing multiple things but it's ok to keep it now, if the function becomes longer we can break it then.